README.md

```diff
@@ -40,13 +40,14 @@ Some small notes on how I've been writing the app so far:
 * I use [octokit](https://github.com/octokit/octokit.js) to wrap GitHub's API to get information about the `uclaacm` org. So far, all of this is ISR'd :)
 * [WestwoodCSS](https://github.com/uclaacm/WestwoodCSS) is still in early alpha, so:
   * there is no documentation (other than reading the source file)
-  * there is no guarantee of forwards-compatability with new versions of WestwoodCSS
+  * there is no guarantee of forwards*compatability with new versions of WestwoodCSS
 * so far, there is no unified types file; most component types live in the component file, and there is some relevant typing in `util/types.ts`
-* the mapping of event data to human-readable components in `components/GitHubEventAction.tsx` is manually done (and currently, manually typed). I haven't *really* looked into it, but...
+* the mapping of event data to human*readable components in `components/GitHubEventAction.tsx` is manually done (and currently, manually typed). I haven't _really_ looked into it, but...
```

Comment:
Is there a reason you've changed the `-` to `*`?

Reply:
My bad, something weird where when I updated ReadME.md, all `*` turned into `-`, so I ctrl+f for all `-`, and accidentally selected those.